services/movie.py

```diff
@@ -6,35 +7,40 @@
 def get_movies(
     genres_ids: list[int] = None,
     actors_ids: list[int] = None,
+    title: str = None
 ) -> QuerySet:
-    queryset = Movie.objects.all()
+    movie = Movie.objects.all()
 
     if genres_ids:
-        queryset = queryset.filter(genres__id__in=genres_ids)
+        movie = movie.filter(genres__id__in=genres_ids)
 
     if actors_ids:
-        queryset = queryset.filter(actors__id__in=actors_ids)
+        movie = movie.filter(actors__id__in=actors_ids)
 
-    return queryset
+    if title:
+        movie = Movie.objects.filter(title__contains=title)
```

Comment:
The filtering logic for `title` should be applied to the existing `QuerySet` (`movie`) instead of creating a new one. This ensures that all filters are combined correctly. Change `movie = Movie.objects.filter(title__contains=title)` to `movie = movie.filter(title__contains=title)`.